composer.json

```diff
@@ -5,11 +5,11 @@
 
     "require": {
         "jms/metadata": "1.*",
-        "symfony/property-access": "~2.2"
+        "symfony/property-access": ">=2.2"
```

Comment:
better explicitly state when this lib is compatible with several major versions, e.g. `~2.2||~3.0`